ehrql/sandbox.py

```diff
@@ -16,6 +16,7 @@ def run(dummy_tables_path):
     # Overwrite __repr__ methods to display contents of frame/series.
     BaseFrame.__repr__ = lambda self: repr(engine.evaluate(self))
     BaseSeries.__repr__ = lambda self: repr(engine.evaluate(self))
+    DateDifference.__repr__ = lambda self: repr(engine.evaluate(self.days))
```

Comment:
It might be helpful to add the units here to make it more obvious that this is a different kind of value from a simple integer.

You could do that using something like this:
```py
DateDifference.__repr__ = lambda self: repr(
    format_column(engine.evaluate(self.days), "{} days")
)

```

With `format_column` defined as:
```py
from ehrql.query_engines.in_memory_database import apply_function


def format_column(column, template):
    return apply_function(template.format, column)
```